util/datetime.go

```diff
@@ -6,20 +6,32 @@ import (
 )
 
 const (
-	shortFormat string = "2006 Jan 02, 15:04"
+	shortFormat string = "2006 Jan 02, 15:04 UTC"
 )
 
 // ParseDate parses our common date/time strings
 // and returns a time.Time object
 func ParseDate(dateString string) time.Time {
 	// our standard format
 	template := "2006-01-02T15:04:05.000Z"
+	alternativeTemplate := "2006-01-02T15:04:05.000-07:00"
 
 	// normalizing the number of decimal places to 3
-	re := regexp.MustCompile("\\.[0-9]+Z$")
-	dateString = re.ReplaceAllLiteralString(dateString, ".000Z")
+	// and discarding sub-second detail along the way
+	re := regexp.MustCompile("\\.[0-9]+")
+	dateString = re.ReplaceAllLiteralString(dateString, ".000")
 
-	t, _ := time.Parse(template, dateString)
+	// try parsing with several formats
+	t, err := time.Parse(template, dateString)
+	if err == nil {
+		return t
```

Comment:
Minor nitpick. In case it is not a parse error we might want to return an error to the caller. See https://golang.org/pkg/time/#ParseError. I think at the moment there is only a parse error returned in case there is any error, but this would make it more future proof. Up to you.